registry/registry.go

```diff
@@ -13,33 +13,36 @@ limitations under the License.
 
 package registry
 
-// Registry abstracts a types registry which holds types that can be
-// used in a Deployment Manager configurations. A registry root must have
-// a 'types' directory which contains all the available types. Each type
-// then contains version directories which in turn contains all the files
-// necessary for that version of the type.
-// For example a type registry holding two types:
-//   redis v1               (implemented in jinja)
-//   replicatedservice v2   (implemented in python)
-// would have a directory structure like so:
-// /types/redis/v1
-//   redis.jinja
-//   redis.jinja.schema
-// /types/replicatedservice/v2
-//   replicatedservice.python
-//   replicatedservice.python.schema
-
-const TypesDir string = "types"
```

Comment:
Why is the prefix being removed? From the documentation it's being changed from types to templates, but removing this seems like we're not having the templates be part of the path.